### DISCUSSION TITLE
Dashboard favourites for every user / team

### DISCUSSION THREAD

**Discussion by PhilippDehner** [grafana/grafana#38868] [Dashboards, Panels & Visualizations]
I've got the problem that the users in my grafana can't find their most used dashboard in an easy way. Therefore my consideration would be to create the possibility to mark dashboards as favourites. Instead of the current way, every user should mark more than one dashboard as favourite. 
So every user can see all Dashboards in the dashboard list or click on a fav-button to filter by favourite dashbaords.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
